### DISCUSSION TITLE
Raspberrymatic as Homeassistant-Addon kills zigbee2mqtt addon

### DISCUSSION THREAD

**Discussion by matmoeh** [OpenCCU/OpenCCU#2226] [🇺🇸 Q&A] [answered]
Hi,

after moving my raspberrymatic to HomeAssistant, i noticed that my zigbee devices arent working.

Starting raspberrymatic addon after zigbee2mqqt: zigbee stops working. In zigbee2mqtt it shows: sendZclFrameToEndpointInternal error
Starting zigbee2mqtt addon after raspberrymatic addon already startet: zigbee2mqtt does not boot. Opening in homeassistant shows: 502: Bad Gateway

Homeassistant Supervised is installed in Proxmox VM hosted on an Intel-NUC.

RPI-RF-MOD Is used over HB-RF-ETH, 5 meters away from Intel-NUC.
SkyConnect is conected with the USB-extension cord

Maybe raspberrymatic are trying to use/connect/break the USB Connection of the SkyConnect USB-Stick?

In the zigbee2mqtt addon the log stops at: Zigbee2MQTT: Starting zigbee-herdsman (0.14.89)
... then it freezes

The next log line should be the info about the coordinator/USB-Stick
Zigbee2MQTT:info  2023-02-19 21:49:46: Coordinator firmware version: '{"meta":{"maintrel":"1 ","majorrel":"7","minorrel":"1","product":9,"revision":"7.1.1.0 build 273"},"type":"EZSP v9"}'

Is there a way to disable some usb features in raspberrymatic to prevent searching and connecting to RF-USB sticks?

**Comment by jens-maus** [comment, marked as answer]
Isn't this exactly the same like https://github.com/jens-maus/RaspberryMatic/issues/2225 ? Actually when closing this issue I routed you to https://github.com/jens-maus/RaspberryMatic/discussions/2182 where you should be able to find the potential root cause: The CUxD Addon taking over the USB stick when RaspberryMatic is started. Thus, disable this CUxD feature by setting `TTYPARAM=NONE` in the CUxD config.

**Comment by matmoeh** [comment]
Oh boy, you are right!
Solution: "disable this CUxD feature by setting TTYPARAM=NONE in the CUxD config."

I was irritated by the correct marked answer and the next reply from you, talking about RF problems and the distance of zigbee to homematic. Thats beacuse i added this information in my post. The correct int was hidden under "view the full answer":

![image](https://user-images.githubusercontent.com/11755384/219977280-a3507072-692e-4d0f-ada3-257b53446a18.png)

I apologize and thank you!!
Paypal support incoming 👍

**Reply by RalphDiesinger** [reply]
Hello
I also have the problem that Zigbee2MQTT no longer starts after installation.
I installed the CUXd Deamon and inserted the corresponding line.
Unfortunately, this does not help. What else can I do?

**Reply by RalphDiesinger** [reply]
Hello
I have now been able to solve the problem myself. It was a misconfiguration of the USB sticks. 
After the change, everything is now possible.
